src/SkedifyAPI.js

```diff
@@ -1,55 +1,40 @@
-// import createCollection from './createCollection'
-// import createIdentityProvider from './IdentityProvider'
+import {
+  applyDecorators,
+  withConfig,
+  withDefaults,
+  withExposedIncludes,
+  withIdentityProvider,
+  withResources,
+  withShorthands,
+} from './decorators'
 
-// export default class SkedifyAPI {
-//   constructor(idp) {
-//     const identityProvider = createIdentityProvider(idp)
+import * as exported from './exported'
```

Comment:
The reason why it is "exported" is because those are the constants that are exported on the API e.g.: `Skedify.API.ERROR_RESOURCE`.
The advantage is that all constants that are used of the codebase are inside the ./constants folder but only the ones in exported are exposed  to the public.

Reply:
Aha! thanks for clearing that up

Reply:
Now that I've added the 'get' and 'post' constants it is a bit more clear now. 

https://github.com/skedify/sdk-js/pull/2/commits/7cdeee557b32c0a67a5365a54baca104ad1c6fec